0.15.2-osx-signed/sjors/bitcoin-dmg-signer-build.assert

```diff
@@ -0,0 +1,548 @@
+--- !omap
+- out_manifest: |
+    5e4a55825dda74fdcab47dcbad61e3984d57a1d464dfc44f242d427b7ba8171d  bitcoin-osx-signed.dmg
```

Comment:
The hash differs from others one.

Reply:
That's odd. My Windows build was causing difficulties, but macOS went fine. I then rebuilt the base image, but only redid the Windows build. Let me also redo the macOS build...